documentation/writing_plans.md

```diff
@@ -173,6 +173,75 @@ parameterized `Sensitive` type, such as `Sensitive` or `Sensitive[Hash]`. Other
 types, such as `Optional[Sensitive]` or `Hash[String, Sensitive]`, will not be
 automatically masked.
 
+## Using Hiera data in plans
+
+You can also use the Puppet `lookup()` function in plans to lookup Hiera data. It's useful to think
+of looking up Hiera data in Bolt plans in two different contexts: inside [apply
+blocks](applying_manifest_blocks.md), and outside apply blocks. Inside apply blocks Bolt is
+compiling Puppet catalogs in a per-target context, and has unfettered access to Hiera data. You can
+use the same Hiera config and data you would for Puppet, and `lookup()` data as you would expect
```

Comment:
```suggestion
use the same Hiera config and data you would for Puppet, and look up data as you would expect
```

Reply:
Oo good call - changed this a few other places as well.